package.json

```diff
@@ -53,7 +53,7 @@
 		"yarn": "^1.22.10"
 	},
 	"scripts": {
-		"postinstall": "npm ci --prefix gutenberg && cd jetpack/projects/plugins/jetpack && yarn install --frozen-lockfile --ignore-engines",
+		"postinstall": "npm ci --prefix gutenberg && cd jetpack/projects/plugins/jetpack && npx pnpm install",
```

Comment:
>Running `pnpm` directly via `npx`. I tried a few different ways to install it on CI but failed. Maybe there's still a way but I mostly given up and went with npx. Happy to revise. Long story short, resolving the module from inside the .pnpm folder since the normally "exposed" paths are actually symlinks and Metro won't work OK with symlinks.

Is there a reason to avoid `npx` from your perspective? Or is it merely a desire to use the "default" flow of `npm install` package and `npm <binary>`?

My only alternative thought would be to explore [relying upon `npm bin`](https://www.rockyourcode.com/run-locally-installed-npm-packages-without-global-install/) to locate the `pnpm` executable (e.g. `$(npm bin)/pnpm install`). Although, I'm not sure how relevant that is to the issues you were seeing installing `pnpm` on CI.

Reply:
> Is there a reason to avoid npx from your perspective? Or is it merely a desire to use the "default" flow of npm install package and npm <binary>?

The latter, I'd intuitively prefer to use something like `pnpm install` with `pnpm` already installed like we had with `yarn`. Also, since this is a "postinstall" phase, feels weird to have to _install_ something here. Not sure if that's a antipattern.

> My only alternative thought would be to explore relying upon npm bin to locate the pnpm executable (e.g. $(npm bin)/pnpm install). Although, I'm not sure how relevant that is to the issues you were seeing installing pnpm on CI.

Aha, that could be an option to try. At this stage, we can probably start with `npx ...` since we don't really need pnpm elsewhere, if you also are satisfied with the current approach too. We should revise if we found ourselves needing it normally installed. WDYT?

Reply:
> Aha, that could be an option to try. At this stage, we can probably start with `npx ...` since we don't really need pnpm elsewhere, if you also are satisfied with the current approach too. We should revise if we found ourselves needing it normally installed. WDYT?

Agreed. I think we should move forward with `npx`. 👍🏻